engine/include/cubos/engine/gizmos/plugin.hpp

```diff
@@ -0,0 +1,26 @@
+/// @dir
+/// @brief @ref gizmos-plugin plugin directory.
+
+/// @file
+/// @brief Plugin entry point.
+/// @ingroup gizmos-plugin
+
+#pragma once
+
+#include <cubos/engine/cubos.hpp>
+#include <cubos/engine/gizmos/gizmos.hpp>
+
+namespace cubos::engine
+{
+    /// @defgroup gizmos-plugin Gizmos
+    /// @ingroup engine
+    /// @brief Used to draw gizmos helpful for debugging and tools.
+    ///
+    /// ## Resources
+    /// - @ref Gizmos - used to queue gizmo draw commands.
```

Comment:
Missing documentation for the tags and plugin deps, checkout other plugins for examples